src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineViewModel_Utilities.cs

```diff
@@ -156,7 +155,11 @@ DocumentSymbolData NestDescendantSymbols(ImmutableArray<RoslynDocumentSymbol> al
 
             // Returns whether the child symbol is in range of the parent symbol.
             static bool Contains(LspDocumentSymbol parent, LspDocumentSymbol child)
-                => child.Range.Start.Line > parent.Range.Start.Line && child.Range.End.Line < parent.Range.End.Line;
+            {
+                var parentRange = ProtocolConversions.RangeToLinePositionSpan(parent.Range);
+                var childRange = ProtocolConversions.RangeToLinePositionSpan(child.Range);
+                return childRange.Start > parentRange.Start && childRange.End <= parentRange.End;
```

Comment:
📝 This is a move from comparing line number to comparing position number. Since the LSP `Position` type didn't support the comparison, we convert to a Roslyn `TextPosition` which provides the `>` and `<=` operators.

Reply:
oh jeez.  yeah.